tests/conjectures_tests.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from ramanujan.LHSHashTable import LHSHashTable
+from ramanujan.enumerators.EfficientGCFEnumerator import EfficientGCFEnumerator
+from ramanujan.poly_domains.CartesianProductPolyDomain import CartesianProductPolyDomain
+from ramanujan.poly_domains.Zeta3Domain1 import Zeta3Domain1
+from ramanujan.constants import g_const_dict
+
+
+def get_testable_data(refined_res_list):
+    '''
+        On those test we have no use for the bloom key or lhs dict pointers
+        exctact the meaning full data for testing
+    '''
+    data = []
+    for i in refined_res_list:
+        data.append((
+            i.rhs_an_poly,
+            i.rhs_bn_poly,
+            i.c_top,
+            i.c_bot))
+    return data
+
+
+class APITests(unittest.TestCase):
+
+    def test_MITM_api1(self):
+        saved_hash = 'e_lhs_dept5_db'
+        lhs_search_limit = 5
+        lhs = LHSHashTable(saved_hash, lhs_search_limit, [g_const_dict['e']]) 
+
+        poly_search_domain = CartesianProductPolyDomain(
+            2, [-5, 5],
+            2, [-5, 5])
+
+        enumerator = EfficientGCFEnumerator(lhs, poly_search_domain, [g_const_dict['e']])
+
+        results = get_testable_data(enumerator.full_execution())
+
+        self.assertEqual(len(results), 42)
```

Comment:
this should make you very suspicious that suddenly we get 42 results...

Reply:
most of them are duplicates caused by not scanning negative an-s
the rest are new gcf found since we now allow some of an's coefs to be negative, so an=2n-1  is now a valid option

regarding pi difference in results, the old tests used b0 as the first object, the results take a different form.
When regarding b1 as the first item, a lot of series are now valid, since the first item is not 0. This leads to plenty of new results.
There a couple of new results i have no good explanation for. ill talk to you about them tomorrow